SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/BaseLink.h

```diff
@@ -150,17 +147,8 @@ class SOFA_CORE_API BaseLink
     /// Print the value of the associated variable
     virtual std::string getValueString() const;
 
-    /// Print the value type of the associated variable
-    virtual std::string getValueTypeString() const;
```

Comment:
Removing this is not correct...use the NameDecoder implementation instead of reimplementing one.